src/admin/services/Validation.ts

```diff
@@ -62,7 +62,19 @@ const additionalRules : IValidationRules = {
 		error: 'Host name is too short',
 	},
 
-	feedbackEmailOverride: {
+	feedback: {
+		type: 'string',
+		exec: customValidations.isEmail,
+		error: 'User mail should be valid email address!',
+	},
+
+	escalation: {
+		type: 'string',
+		exec: customValidations.isEmail,
+		error: 'User mail should be valid email address!',
+	},
+
+	fallback: {
```

Comment:
is it possible to make one rule  - "email"  and do not repeat the same code three times?